desktop/libs/azure/src/azure/abfs/upload.py

```diff
@@ -35,10 +35,89 @@
 else:
   from django.utils.translation import ugettext as _
 
-DEFAULT_WRITE_SIZE = 30 * 1000 * 1000 # TODO: set in configuration
+DEFAULT_WRITE_SIZE = 128 * 1000 * 1000 # TODO: set in configuration
```

Comment:
Change the value on the comment also.

Also maybe set this as a config like said in the comment too?